cli/command/formatter/node.go

```diff
@@ -12,7 +14,7 @@ import (
 )
 
 const (
-	defaultNodeTableFormat           = "table {{.ID}} {{if .Self}}*{{else}} {{ end }}\t{{.Hostname}}\t{{.Status}}\t{{.Availability}}\t{{.ManagerStatus}}"
+	defaultNodeTableFormat           = "table {{.ID}} {{if .Self}}*{{else}} {{ end }}\t{{.Hostname}}\t{{.Status}}\t{{.Availability}}\t{{.ManagerStatus}}{{if .TLSStatus}}\t{{.TLSStatus}}{{end}}"
```

Comment:
I was thinking to show it in `docker node ls`, so if the root rotation is hanging, we can see which node(s) may be problematic and not able to renew.